.github/workflows/cloud_regression.yml

```diff
@@ -22,21 +22,24 @@ jobs:
           PR_COMMIT_HASH: ${{ github.event.pull_request.head.sha }}
         id: output-workflow-dispatch-params
         run: |
-          if [ ${{ github.event_name }} == 'pull_request_target' ]; then
+          if [ "${{ github.event_name }}" == 'pull_request_target' ]; then
             NETDATA_CUSTOM_REPO="$PR_REPO_NAME"
             NETDATA_CUSTOM_BRANCH="$PR_BRANCH_NAME"
             NETDATA_CUSTOM_PR_NUMBER="${{ github.event.number }}"
             NETDATA_CUSTOM_COMMIT_HASH="$PR_COMMIT_HASH"
-          elif [ ${{ github.event_name }} == 'push' ]; then
+          elif [ "${{ github.event_name }}" == 'push' ]; then
             NETDATA_CUSTOM_REPO="netdata/netdata"
             NETDATA_CUSTOM_BRANCH="master"
             NETDATA_CUSTOM_PR_NUMBER=""
             NETDATA_CUSTOM_COMMIT_HASH="${{ github.sha }}"
           fi
-          echo "netdata_repo=${NETDATA_CUSTOM_REPO}" >> $GITHUB_OUTPUT
-          echo "netdata_branch=${NETDATA_CUSTOM_BRANCH}" >> $GITHUB_OUTPUT
-          echo "netdata_pr_number=${NETDATA_CUSTOM_PR_NUMBER}" >> $GITHUB_OUTPUT
-          echo "netdata_commit_hash=${NETDATA_CUSTOM_COMMIT_HASH}" >> $GITHUB_OUTPUT
+
+          {
+            echo "netdata_repo=${NETDATA_CUSTOM_REPO}"
+            echo "netdata_branch=${NETDATA_CUSTOM_BRANCH}"
+            echo "netdata_pr_number=${NETDATA_CUSTOM_PR_NUMBER}"
+            echo "netdata_commit_hash=${NETDATA_CUSTOM_COMMIT_HASH}"
+          } >> "$GITHUB_OUTPUT"
```

Comment:
Have we confirmed that this is semantically equivalent from _GitHub’s_ perspective? The general idea is great, but the original was written the way it was because I have been unable to find any conclusive evidence regarding whether the runner environment cares how many times the file is opened/closed or not.

Reply:
TTBMOK yes this is acceptable. The side effect of merging this change won't be the end of the world. We can keep an eye on this workflow in the upcoming days.